inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -92,6 +92,10 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
             if (absoluteDifference <= threshold) {
                 continue;
             }
+            if (std::isnan(static_cast<float>(res)) &&
```

Comment:
Looks like we have the same condition in the line: 107. I think this code should be changed,

Reply:
Thanks @ilyachur, I just check latest code:https://github.com/openvinotoolkit/openvino/blob/bba7155fb722b72b22ca684d451490b0151c6405/inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp#L107

It doesn't process condition: **ref and res both are nan.**

Reply:
I mean, that looks like we can change the logic to avoid the same conditions which are different only by logical operand (`||` or `&&`).

Reply:
Thanks @ilyachur for quick clarification. 
The previous comment (_Please rebase your PR to the latest master_.) confuse me.

Do you think I can update it in this PR? or do we need to create a new JIRA and then create a new PR?

Reply:
I think we can fix it in this PR without additional Jira tickets 😄

Reply:
🤩， I just updated it again, it looks like better, @ilyachur Please help review it again. Thanks.

-std::isnan(static_cast<float>(res)) || std::isnan(static_cast<float>(ref))) 
+(std::isnan(static_cast<float>(res)) ^ std::isnan(static_cast<float>(ref))))